common/lib/xmodule/xmodule/fields.py

```diff
@@ -252,3 +253,19 @@ def enforce_type(self, value):
             return value
 
         return self.from_json(value)
+
+
+class ScoreField(JSONField):
+    def from_json(self, value):
+        if value is None:
+            return value
+        if isinstance(value, Score):
+            return value
```

Comment:
I'll re-run tests on the branch I opened. https://github.com/edx/edx-platform/pull/15452